src/components/content/order/common/BillingInfo.tsx

```diff
@@ -18,7 +18,7 @@ export const BillingInfo = ({ priceValue, billing }: { priceValue: string; billi
                 </Tag>
                 <span className={'order-billing-model-title-class'}>Model:&nbsp;</span>
                 <Tag color={'blue'} className={'order-billing-model-value-class'}>
-                    {billing.billingModel}
+                    {'monthly'}
```

Comment:
can we simply take the first billing instead of this fixed value. We will have to anyways fully implement it in other userstory. 
Also please add a TODO comment.